client/components/jetpack-colophon/docs/example.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import JetpackColophon from 'components/jetpack-colophon';
+
+export default function JetpackColophonExample() {
+	return (
+		<div style={ { fontSize: '15px' } }>
+			<JetpackColophon />
+		</div>
+	);
+}
+JetpackColophonExample.displayName = 'JetpackColophonExample';
```

Comment:
The `displayName` should be `JetpackColophon`, right?

Reply:
This is actually the convention:

https://github.com/Automattic/wp-calypso/blob/master/client/devdocs/design/README.md#component-name-convention

![componentexample](https://user-images.githubusercontent.com/841763/30907008-d41e65ce-a379-11e7-8171-730f842562d9.png)

Reply:
Ha, TIL! 👍 

Well, we'll probably want to update that here and there then.

Reply:
Yeah, lots of existing violations 🤷‍♂️